docs/dev/install.rst

```diff
@@ -50,7 +50,7 @@ Installation
 
    .. code-block:: bash
 
-      pip install -r requirements/default.txt
+      pip install -r requirements/default.txt -c requirements/constraints.txt
```

Comment:
This isn't needed, since the first line in `requirements/default.txt` is `-c constraints.txt`. That causes pip to use the constraints file with less typing.

Reply:
Neat! I totally missed that.